src/RadarScatter.js

```diff
@@ -266,7 +265,6 @@ var RadarScatter = function() {
                 // Customize the chart for being tiny!
                 .lineThickness(0.5)
                 .showVertice(false)
-                .showTooltip(false)
```

Comment:
So if you look at the vertice part, I will only show hover message when showVertiece is true which is in the large-radar div. Does it make sense?